src/foxlab/key65_hotswap/key65_hotswap.json

```diff
@@ -0,0 +1,175 @@
+{
+  "name": "Key 65 Universal",
+  "vendorId": "0x464C",
+  "productId": "0x0003",
+  "lighting": "qmk_backlight_rgblight",
+  "matrix": {"rows": 5, "cols": 15},
+  "layouts": {"labels": [
+    "Top Blocker",
+    "Windows Key"
+  ],
+  "keymap": [
+    [
+      {
+        "y": 1.5,
+        "x": 2.5,
+        "c": "#777777"
+      },
+      "0,0",
+      {
+        "c": "#cccccc"
+      },
+      "0,1",
+      "0,2",
+      "0,3",
+      "0,4",
+      "0,5",
+      "0,6",
+      "0,7",
+      "0,8",
+      "0,9",
+      "0,10",
+      "0,11",
+      "0,12",
+      {
+        "c": "#aaaaaa",
+        "w": 2
+      },
+      "0,13",
+      "0,14\n\n\n0,0",
+      {
+        "x": 0.5,
+        "d": true
+      },
+      "0,14\n\n\n0,1"
+    ],
+    [
+      {
+        "x": 2.5,
+        "w": 1.5
+      },
+      "1,0",
+      {
+        "c": "#cccccc"
+      },
+      "1,1",
+      "1,2",
+      "1,3",
+      "1,4",
+      "1,5",
+      "1,6",
+      "1,7",
+      "1,8",
+      "1,9",
+      "1,10",
+      "1,11",
+      "1,12",
+      {
+        "w": 1.5
+      },
+      "1,13",
+      {
+        "c": "#aaaaaa"
+      },
+      "1,14"
+    ],
+    [
+      {
+        "x": 2.5,
+        "w": 1.75
+      },
+      "2,0",
+      {
+        "c": "#cccccc"
+      },
+      "2,1",
+      "2,2",
+      "2,3",
+      "2,4",
+      "2,5",
+      "2,6",
+      "2,7",
+      "2,8",
+      "2,9",
+      "2,10",
+      "2,11",
+      {
+        "c": "#777777",
+        "w": 2.25
+      },
+      "2,12",
+      {
+        "c": "#aaaaaa"
+      },
+      "2,14"
+    ],
+    [
+      {
+        "x": 2.5,
+        "w": 2.25
+      },
+      "3,0",
+      {
+        "c": "#cccccc"
+      },
+      "3,2",
+      "3,3",
+      "3,4",
+      "3,5",
+      "3,6",
+      "3,7",
+      "3,8",
+      "3,9",
+      "3,10",
+      "3,11",
+      {
+        "c": "#aaaaaa",
+        "w": 1.75
+      },
+      "3,12",
+      "3,13",
+      "3,14"
+    ],
+    [
```

Comment:
An aesthetic thing, but the bottom row is wrong. Should be 1.5/1/1.5 rather than 1.5/1.25/1.5. [See here.](http://www.keyboard-layout-editor.com/##@@_y:1.5&x:2.5&c=%23777777%3B&=0,0&_c=%23cccccc%3B&=0,1&=0,2&=0,3&=0,4&=0,5&=0,6&=0,7&=0,8&=0,9&=0,10&=0,11&=0,12&_c=%23aaaaaa&w:2%3B&=0,13&=0,14%0A%0A%0A0,0&_x:0.5&d:true%3B&=0,14%0A%0A%0A0,1%3B&@_x:2.5&w:1.5%3B&=1,0&_c=%23cccccc%3B&=1,1&=1,2&=1,3&=1,4&=1,5&=1,6&=1,7&=1,8&=1,9&=1,10&=1,11&=1,12&_w:1.5%3B&=1,13&_c=%23aaaaaa%3B&=1,14%3B&@_x:2.5&w:1.75%3B&=2,0&_c=%23cccccc%3B&=2,1&=2,2&=2,3&=2,4&=2,5&=2,6&=2,7&=2,8&=2,9&=2,10&=2,11&_c=%23777777&w:2.25%3B&=2,12&_c=%23aaaaaa%3B&=2,14%3B&@_x:2.5&w:2.25%3B&=3,0&_c=%23cccccc%3B&=3,2&=3,3&=3,4&=3,5&=3,6&=3,7&=3,8&=3,9&=3,10&=3,11&_c=%23aaaaaa&w:1.75%3B&=3,12&=3,13&=3,14%3B&@_x:2.5&w:1.5%3B&=4,0&_d:true%3B&=4,1%0A%0A%0A1,0&_w:1.5%3B&=4,2&_c=%23cccccc&w:7%3B&=4,7&_c=%23aaaaaa&w:1.5%3B&=4,11&_x:0.5%3B&=4,12&=4,13&=4,14%3B&@_y:0.5&x:4%3B&=4,1%0A%0A%0A1,1)

Reply:
Oh I had actually just noticed this, didn't see it before. Though I updated it for all the bottom rows and fixed it up. I don't know if I should have added a separate WKL option for each bottom row but I did it anyway.